ext/openssl/xp_ssl.c

```diff
@@ -58,18 +58,20 @@
 #define STREAM_CRYPTO_METHOD_TLSv1_0       (1<<3)
 #define STREAM_CRYPTO_METHOD_TLSv1_1       (1<<4)
 #define STREAM_CRYPTO_METHOD_TLSv1_2       (1<<5)
+#define STREAM_CRYPTO_METHOD_TLSv1_3       (1<<6)
 
 #ifndef OPENSSL_NO_SSL3
 #define HAVE_SSL3 1
 #define PHP_OPENSSL_MIN_PROTO_VERSION STREAM_CRYPTO_METHOD_SSLv3
 #else
 #define PHP_OPENSSL_MIN_PROTO_VERSION STREAM_CRYPTO_METHOD_TLSv1_0
 #endif
-#define PHP_OPENSSL_MAX_PROTO_VERSION STREAM_CRYPTO_METHOD_TLSv1_2
+#define PHP_OPENSSL_MAX_PROTO_VERSION STREAM_CRYPTO_METHOD_TLSv1_3
 
 
 #define HAVE_TLS11 1
 #define HAVE_TLS12 1
+#define HAVE_TLS13 1
```

Comment:
this is not going to work for older OpenSSL versions so it needs to check the OpenSSL version